Makefile

```diff
@@ -17,14 +17,17 @@ lint: format
 	poetry run flake8 aws_lambda_powertools/* tests/*
 
 test:
-	poetry run pytest -m "not perf" --cov=aws_lambda_powertools --cov-report=xml
+	poetry run pytest -m "not (perf or e2e)" --cov=aws_lambda_powertools --cov-report=xml
 	poetry run pytest --cache-clear tests/performance
 
 unit-test:
 	poetry run pytest tests/unit
 
+e2e-test:
+	poetry run pytest -n 3 --dist loadscope --durations=0 --durations-min=1 tests/e2e
+
 coverage-html:
-	poetry run pytest -m "not perf" --cov=aws_lambda_powertools --cov-report=html
+	poetry run pytest -m "not (perf or e2e)" --cov=aws_lambda_powertools --cov-report=html
```

Comment:
Same as above, ```-m "not (perf or e2e)"``` doesn't work as expected as the e2e tests aren't marked

Reply:
all e2e tests are marked with `@pytest.mark.e2e`